src/client/component/dedicated_info.cpp

```diff
@@ -17,17 +17,18 @@ namespace dedicated_info
 		{
 			const auto sv_running = game::Dvar_FindVar("sv_running");
 			const auto server_name = game::get_dvar_string("live_steam_server_name");
```

Comment:
just remove the const here :D
```suggestion
			auto server_name = game::get_dvar_string("live_steam_server_name");
```

Reply:
these suggestions are super nice 😂